pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -625,6 +635,8 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				// TODO: change glusterfs to use DNS lookup so this isn't needed?
 				// Needed for glusterfs volumes
 				authorizationapi.NewRule("get").Groups(kapiGroup).Resources("endpoints").RuleOrDie(),
+				// Needed for Dynamic Provisioning
+				authorizationapi.NewRule("get").Groups(storageGroup).Resources("storageclasses").RuleOrDie(),
```

Comment:
surprised unit tests didn't complain about this permission not being in the test fixtures